rocky/rocky/middleware/onboarding.py

```diff
@@ -24,12 +24,15 @@ def middleware(request):
                 or request.path.startswith("/api/")
             ):
                 if not member_onboarded:
-                    if is_red_team(request.user):
+                    member = OrganizationMember.objects.filter(user=request.user)
+
+                    # There might be redteamers without an organization after an organization is deleted.
+                    if member.exists() and is_red_team(request.user):  #
```

Comment:
```suggestion
                    if member.exists() and is_red_team(request.user):
```

Reply:
This is not just limited to red-teamers. Admins and clients are mostly the same.

Reply:
@Rieven Was there a specific reason to filter on redteamers here?

Reply:
> @Rieven Was there a specific reason to filter on redteamers here?

@Donnype Yes, when member is not onboarded at startup it is redirected to the redteamers onboarding flow/views.

Reply:
Answer: yes clients do not necessarily need to be onboarded(?)